src/App.tsx

```diff
@@ -1,11 +1,40 @@
-import React from 'react'
 import '@/App.css'
-import LoginPage from '@/pages/LoginPage/LoginPage'
+import EmployeesPage from '@/pages/EmployeesPage/EmployeesPage'
+import { Breadcrumb, Layout, Menu } from 'antd'
+import React from 'react'
+import MainSidepanel from './components/UI/sidepanels/MainSidepanel/MainSidepanel'
 
 function App() {
+  const { Header, Content, Sider } = Layout
+
   return (
     <div className='App'>
-      <LoginPage />
+      <Layout>
```

Comment:
No, it should be another way, you need to create router and after it, provide you Employee component. It would be wonderful if you create some kind of layout with header and footer to reduce code dupes